src/score_models/sbm/score_model.py

```diff
@@ -67,22 +72,19 @@ def log_likelihood(
         See Song et al. 2020 (arxiv.org/abs/2011.13456) for usage with SDE formalism of SBM.
         """
         B, *D = x.shape
-
         solver = ODESolver(self, solver=solver, **kwargs)
         # Solve the probability flow ODE up in temperature to time t=1.
-        xT, dlog_p = solver(
-            x, *args, steps=steps, forward=True, t_min=t, **kwargs, get_delta_logp=True
+        xT, dlogp = solver(
+            x, *args, steps=steps, forward=True, t_min=t, **kwargs, return_logp=True, dlogp=self.dlogp
```

Comment:
not sure about `return_logp` since this does not return the logp, it returns the change in logp due to traversing the SDE. It does not include the logp of the terminal distribution, which is added a couple lines down